Tests/modules/io_related/test_codecs.py

```diff
@@ -21,9 +21,12 @@
 import sys
 import unittest
 
-from iptest import IronPythonTestCase, is_cli, is_cpython, is_mono, is_netcoreapp, is_posix, run_test, skipUnlessIronPython
+from iptest import IronPythonTestCase, is_cli, is_cpython, is_mono, is_netcoreapp, is_posix, is_windows, is_osx, run_test, skipUnlessIronPython
 from iptest.misc_util import ip_supported_encodings
 
+if is_cpython and is_posix:
```

Comment:
I was a bit quick when merging in https://github.com/IronLanguages/ironpython3/pull/699. `is_posix` should actually mean mac or linux. I'll hold off on merging this until https://github.com/IronLanguages/ironpython3/issues/704 is done and amend this PR to use `is_linux` instead.

Reply:
If `is_posix` means mac or linux, then the whole mac commit  e6a53913ce0677fd726394fa1adb17f43eaa1907 can/should be reverted.

Reply:
I've merged in the changes, may be easier for you to roll back the commit. Otherwise I'll try to do it later today.

Reply:
OK, I will do it now.